subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/FileSystemWatchingPerformanceTest.groovy

```diff
@@ -43,7 +43,7 @@ class FileSystemWatchingPerformanceTest extends AbstractCrossVersionPerformanceT
     private static final String KOTLIN_TARGET_VERSION = new KotlinGradlePluginVersions().latests.last()
 
     def setup() {
-        runner.minimumBaseVersion = "6.7"
+        runner.minimumBaseVersion = "7.0"
```

Comment:
Does this mean that we won't be comparing pre-7.0 performance? It'd be great if we could keep that.

Reply:
I reverted the requirements. Note that this is a performance regression test, which doesn't test any pre-7.0 performance anyway. Only for the historical tests which run once a week, but even there we have the older 6.x data in the graph.